nav2_costmap_2d/include/nav2_costmap_2d/inflation_layer.hpp

```diff
@@ -108,21 +102,21 @@ class InflationLayer : public Layer
   inline unsigned char computeCost(double distance) const
   {
     unsigned char cost = 0;
+    double euclidean_distance = distance * resolution_;
     if (distance == 0) {
       cost = LETHAL_OBSTACLE;
-    } else if (distance * resolution_ <= inscribed_radius_) {
+    } else if (euclidean_distance <= inscribed_radius_) {
```

Comment:
Please revert this, you do an extra copy in that change + the variable isn't later re-used.